_posts/2015-06-16-hard-fork-policy.md

```diff
@@ -0,0 +1,41 @@
+---
+type: posts
+layout: post
+lang: en
+category: blog
+
+title: "Bitcoin.org Hard Fork Policy"
+permalink: /en/posts/hard-fork-policy.html
+date: 2015-06-16
+---
+It appears that the recent block size debate will likely result in a
+contentious hard fork attempt.
+
+Contentious hard forks are bad for Bitcoin. At the very best, a
+contentious hard fork will leave people who chose the losing side of the
+fork feeling disenfranchised. At the very worst, it will make bitcoins
+permanently lose their value. In between are many possible outcomes, but
+none of them is good.
+
+The danger of a contentious hard fork is potentially so significant
+that Bitcoin.org has decided to adopt a new policy:
+
+> Bitcoin.org will not promote software or services that will leave the
+> previous consensus because of a contentious hard fork attempt.
```

Comment:
I think replacing "contentious" with "contentious, intentional" might be better. Accidental hard forks do happen, it is is possible that their resolution could be contentious.